setup.cfg

```diff
@@ -54,7 +54,7 @@ tests =
     invenio-db[postgresql,mysql,versioning]>=1.0.14,<2.0.0
     pytest-invenio>=2.1.0,<3.0.0
     pytest-mock>=1.6.0
-    sphinx>=4.2.0,<5
+    sphinx>=5,<6
```

Comment:
I had to update this in order for the tests to pass, the error was caused by one of the extensions
![Screenshot from 2024-01-15 15-07-05](https://github.com/inveniosoftware/invenio-records-resources/assets/15194802/408111c2-7f1e-483c-b0da-9f7bdf239ee7)
Apparently it was introduced some time ago but there was a release recently that maybe it's being picked up now, see here [changelog](https://github.com/sphinx-doc/sphinxcontrib-applehelp/blob/master/CHANGES#L22C47-L22C47).
The other alternative is to pin the extensions to <1.0.5